src/vscode/commands/Repl.ts

```diff
@@ -19,6 +19,10 @@ import {
     useEditor,
     useRepl,
 } from '../Utils'
+import * as childProcess from 'child_process';
+
+const outputChannel = vscode.window.createOutputChannel("Alive")
+let child: childProcess.ChildProcessWithoutNullStreams | null
```

Comment:
This needs to be set to null here, or change to using undefined instead of null. It's initially undefined, which gets a little confusing. Not sure why TS doesn't complain about it.